README.md

```diff
@@ -1,2 +1,168 @@
+
+- It shows a random greeting
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>HTML</li>
+   <li>Bootstrap</li>
+   <li>React</li>
+   <li>Redux</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- It shows a random greeting
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Currently, I don't have a live demo of this project. I will let you know when it is available ASAP!
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Getting Started -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To run the project locally, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+
+- Install [Git](https://git-scm.com/)
+- Install [Ruby](https://www.ruby-lang.org/en/)
+- Install Rails gem `gem install rails`
+- Install a code editor, We suggest [VsCode](https://code.visualstudio.com/)
+
+### Setup
+
+Clone the repo into your local machine:
+
+```bash
+git clone git@github.com:OsvaldoBC/hello-rails-react.git
+```
+
+Install the dependencies using bundler:
+```bash
+bundle install
+```
+
+### Usage
+
+To start you can open your terminal and in the folder write
+```bash
+ruby main.rb
+```
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤👤 **Osvaldo Barrios**
+
+- Twitter : [@OsvaldoBarrio20](https://twitter.com/OsvaldoBarrio20)
+- GitHub :  [@OsvaldoBC](https://github.com/OsvaldoBC)
+- LinkedIn: [osvaldo-barrios-data-science](https://linkedin.com/in/osvaldo-barrios-data-science)
+<!-- FUTURE FEATURES -->
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Improve UI/UX design
+- Create a Desktop version
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Contributing -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- Show your support -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project leave a start for it.
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse for helping us in the journey to become a Fullstack developer 🌹
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
```

Comment:
- [ ] I think it would be better if you fix the broken `[MIT](./LICENSE)` link in your README file to make it more professional as it shows a **404** error page once clicked. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same content in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**
![Screenshot from 2023-07-20 03-12-51](https://github.com/OsvaldoBC/hello-rails-react/assets/83042742/2d1bd3a6-fdf5-4a18-9395-73286bc46681)